apps/tehwolfde/src/assets/fonts/fonts.css

```
@@ -0,0 +1,78 @@
+/* latin-ext */
+@font-face {
+  font-family: 'Roboto';
```

Stylelint (17.13.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)